docs/install/terminal/README.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Terminal
+---
+
+# IPFS terminal
```

Comment:
What is the terminal deamon? Can we remove this rearchitecting of the install flow from this update? I think it adds unnecessary complexity...

Reply:
I understood the IPFS daemon to be what you get when you run `ipfs daemon` through a terminal. I've removed the _daemon_ wording from this section: 8ac058b

> Can we remove this rearchitecting of the install flow from this update?

So you'd like to remove the `/install/ipfs-companion`, `/install/ipfs-desktop`, and `/install/terminal` sections?

Reply:
so "install/terminal" is the previous "install go-ipfs" - right? I wouldn't understand that from the title "terminal" - which is a problem because now I need to know to click on that header instead of just scrolling through a single page

Reply:
I think the information architecture of this list with "desktop" and "browser" is unclear, and I'm concerned this isn't going to get people to install the right set of things (ideally, we'd have 3 easy buttons that install go-ipfs, IPFS Desktop, and IPFS Companion)

Reply:
Ah right, I see what you mean now. Commit 048a153 fixes this issue. We can discuss how to map out the install instructions at some other point, like you say.